packages/react-components/react-theme-sass/src/index.ts

```diff
@@ -0,0 +1,5 @@
+throw new Error(
```

Comment:
There is no Javascript code in this package. It only contains SASS variables.
To avoid any confusion on consumer side, the package throws when imported:
<img width="939" alt="image" src="https://user-images.githubusercontent.com/9615899/169002627-e9929872-d0d5-4823-a81a-d7c6b3127618.png">

Reply:
we discussed this with @miroslavstastny and agreed that to maintain same patterns in our package this is good enough approach without additional maintenance. we can revisit in the future for sure.